backend/dataall/modules/dashboards/services/dashboard_quicksight_service.py

```diff
@@ -103,6 +105,7 @@ def get_monitoring_vpc_connection_id():
 
     @classmethod
     def create_quicksight_data_source_set(cls, vpc_connection_id):
+        cls._check_user_must_be_admin()
```

Comment:
Can we do a decorator out of it? It seems to be a critical check everywhere

Reply:
I'll merge this one and work on the generic decorator. We should also add admin-permission tests in the same way we do with app level permissions and nested resources